projects/openfga/openfga-incubation-dd.md

```diff
@@ -0,0 +1,346 @@
+# OpenFGA Incubation Due Diligence
+
+- Link to [Incubation application issue](https://github.com/cncf/toc/issues/1287)
+
+<!-- This template provides the TOC with the outline for completing due diligence of a project to move levels. This universal template is designed to capture all criteria so the TOC may ensure prior level criteria do not regress. As part of completing the due diligence, the TOC member should update the template to convey the level the project applied for the criteria by bolding the level indicated where the criteria is relevant. -->
+
+## Incubation Evaluation Summary for OpenFGA
+
+### Criteria Evaluation
+
+# Criteria Evaluation
+
+$TOCMember conducted the due diligence of **OpenFGA**, which applied for incubation. The project has <TBD> the criteria that demonstrate its maturity at incubation.
```

Comment:
@kfaseela please update the evaluation section w/ your name and Ricardo A helping with the Adopter interviews. Also, whether the project has or has not met the criteria to move to incubation.